be/src/olap/rowset/beta_rowset_writer_v2.cpp

```diff
@@ -74,13 +74,25 @@ Status BetaRowsetWriterV2::create_file_writer(uint32_t segment_id, io::FileWrite
     auto index_id = _context.index_id;
     auto tablet_id = _context.tablet_id;
     auto load_id = _context.load_id;
-
     auto stream_writer = std::make_unique<io::StreamSinkFileWriter>(_streams);
     stream_writer->init(load_id, partition_id, index_id, tablet_id, segment_id);
     file_writer = std::move(stream_writer);
     return Status::OK();
 }
 
+Status BetaRowsetWriterV2::create_inverted_index_v2_file_writer(uint32_t segment_id,
```

Comment:
You can add a argument type to create_file_writer() to avoid adding a similar function create_inverted_index_v2_file_writer and duplicate code.

Reply:
done